### PR TITLE
【PPSCI Doc No.76】

### PR DESCRIPTION
**Pull request by NKNaN**
### PR types
Others

### PR changes
Docs

### Describe
增加了描述和示例代码，看看这样改是否合适，可以的话其他的loss我也按这样来改吧

### CODE REVIEW

File: ppsci/loss/l1.py [hunk 1]
```diff
@@ -35,13 +35,41 @@ class L1Loss(base.Loss):
     \mathbf{x}, \mathbf{y} \in \mathcal{R}^{N}
     $$
 
+    when `reduction` set to be "mean"
```
Comment (HydrogenSulfate) [inline ppsci/loss/l1.py:38]:
``` suggestion
    when `reduction` is set to "mean"
```

File: ppsci/loss/l1.py [hunk 2]
```diff
@@ -35,13 +35,41 @@ class L1Loss(base.Loss):
...
+
+    $$
+    L = MEAN \left( \Vert \mathbf{x} - \mathbf{y} \Vert_1 \right)
+    $$
+
+    when `reduction` set to be "sum"
```
Comment (HydrogenSulfate) [inline ppsci/loss/l1.py:44]:
``` suggestion
    when `reduction` is set to "sum"
```

File: ppsci/loss/l1.py [hunk 3]
```diff
@@ -35,13 +35,41 @@ class L1Loss(base.Loss):
...
     Examples:
-        >>> import ppsci
-        >>> loss = ppsci.loss.L1Loss()
+        >>> import paddle
+        >>> from ppsci.loss import L1Loss
+        >>> output_dict = {'y1': paddle.to_tensor([[0.5, 0.9], [1.1, -1.3]]),
+        ...                'y2': paddle.to_tensor([[0.5, 0.9], [1.1, -1.3]])}
+        >>> label_dict = {'y1': paddle.to_tensor([[-1.8, 1.0], [-0.2, 2.5]]),
+        ...               'y2': paddle.to_tensor([[0.1, 0.1], [0.1, 0.1]])}
+        >>> weight = {'y1': 0.8, 'y2': 0.2}
```
Comment (HydrogenSulfate) [inline ppsci/loss/l1.py:61]:
``` suggestion
        >>> output_dict = {"u": paddle.to_tensor([[0.5, 0.9], [1.1, -1.3]]),
        ...                "v": paddle.to_tensor([[0.5, 0.9], [1.1, -1.3]])}
        >>> label_dict = {"u": paddle.to_tensor([[-1.8, 1.0], [-0.2, 2.5]]),
        ...               "v": paddle.to_tensor([[0.1, 0.1], [0.1, 0.1]])}
        >>> weight = {"u": 0.8, "v": 0.2}
```

**Review by HydrogenSulfate — APPROVED** [review]
辛苦大佬

**Comment by NKNaN** [comment]
感谢~

**Review by HydrogenSulfate — APPROVED** [review]
LGTM
